pkg/ddl/executor.go

```diff
@@ -6435,6 +6441,21 @@ func (e *executor) DoDDLJobWrapper(ctx sessionctx.Context, jobW *JobWrapper) (re
 	}
 }
 
+func getRenameTableUniqueIDs(job *model.Job, schema bool) []int64 {
+	if !schema {
+		return []int64{job.TableID}
+	}
+
+	var oldSchemaID int64
+	if job.Version <= model.JobVersion1 {
+		oldSchemaID = job.Args[0].(int64)
```

Comment:
the first v1 args is a slice

Reply:
psss it by JobWrapper.JobArgs, so we can directly access it here

Reply:
Done